GSA/digitalgov.gov
# HCD #2: Add Glossary Component

## Summary

Adds the [18F Glossary Component](https://github.com/18F/glossary) to all HCD guide pages. 
> [!WARNING]  
> Requires #6419 to be merged first



### Preview

[Link to Preview](https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.sites.pages.cloud.gov/preview/gsa/digitalgov.gov/jtm-add-hcd-glossary/)

### Solution
<img width="1438" alt="Screenshot 2023-08-04 at 4 00 14 PM" src="https://github.com/GSA/digitalgov.gov/assets/56370370/edf5450e-76fc-480b-8770-dae8598a820d">

### How To Test
1. In the `_index.md` file of any guide, create a parameter called `glossary` and set it equal to `hcd-glossary.json`
2. Click the glossary link in the resulting guide—the HCD glossary should appear

> [!NOTE]  
>  ~~In future iterations, we should look into ways to display more than one glossary. Will require communication between Hugo and Javascript, and may require moving away from 18F component.~~
> 
> This was addressed in [4cc24b7](https://github.com/GSA/digitalgov.gov/pull/6420/commits/4cc24b7589bc4d0e13383e8a2ebb550ac369176e)

---

### Dev Checklist
